src/java/voldemort/server/VoldemortServer.java

```diff
@@ -140,6 +144,16 @@ public VoldemortServer(VoldemortConfig config, Cluster cluster) {
         createOnlineServices();
     }
 
+    private void enableBouncyCastleForSSL(boolean enable) {
+        if (enable) {
+            if (voldemortConfig.isBouncyCastleEnabled()) {
```

Comment:
Can we remove the enable parameter for this method, the same config is read twice.